app/src/main/java/com/platypus/android/tablet/TeleOpPanel.java

```diff
@@ -1010,9 +996,7 @@ public void run() {
             if (currentBoat != null) {
               connected = currentBoat.getConnected();
 
-              //if (currentBoat.getConnected() == true)
-              {
-                if (old_thrust != thrustTemp || old_rudder!=rudderTemp) {
+                //if (old_thrust != thrustTemp || old_rudder!=rudderTemp) {
```

Comment:
Just get rid of these old comments.  I have no idea what this is supposed to be doing but it looks like garbage.